packages/web-components/src/spinner/spinner.options.ts

```diff
@@ -0,0 +1,48 @@
+export type Values<T> = T[keyof T];
+/**
+ * Spinner size options
+ * @public
+ */
+export const SpinnerSize = {
+  /**
+   * A Spinner's size can be small, tiny, extra-small, medium, large, extra-large, or huge
+   *
+   */
+  small: 'small',
+  tiny: 'tiny',
+  extraSmall: 'extra-small',
+  medium: 'medium',
+  large: 'large',
+  extraLarge: 'extra-large',
+  huge: 'huge',
+} as const;
+
+/**
+ * Spinner appearance options
+ * @public
+ */
+export const SpinnerAppearance = {
+  /**
+   * Use the primary appearance when the Spinner is on a light background
+   *
+   */
+  primary: 'primary',
+
+  /**
+   * Use the inverted appearance when the Spinner is on a dark background
+   *
+   */
+  inverted: 'inverted',
+} as const;
+
+/**
+ * The type for Spinner size option
+ * @public
+ */
+export type SpinnerSize = Values<typeof SpinnerSize>;
```

Comment:
reorder: move this just bellow the `SpinnerSize` object to match the order used in another components

Reply:
Sorry, I was not clear enough.
The order should be:

```ts
export const SpinnerAppearance = {};
export type SpinnerAppearance = Values<typeof SpinnerAppearance>;

export const SpinnerSize = {};
export type SpinnerSize = Values<typeof SpinnerSize>;
```